examples/LazyLoadClosestImages.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react";
+import Slider from "../src/slider";
+import { baseUrl } from "./config";
+
+export default class LazyLoadClosestImages extends Component {
+    render() {
+        const settings = {
+            dots: true,
+            lazyLoad: [0, 1],
```

Comment:
could we document this a bit more - I'm not sure what it means just looking at this quickly.
I'm not sure where to document it because this doesn't use our boilerplate conventions.  Maybe just a comment here might be enough and reference it in the changelog

There  are documentation and playground links in the README which point to the old react-slick which might become increasingly out of date - not sure if we should change that here as well.